vitest.base.unit.config.ts

```diff
@@ -20,7 +20,7 @@ export default defineConfig({
     ],
     reporters: process.env.GITHUB_ACTIONS
       ? ["verbose", "hanging-process", "github-actions"]
-      : ["verbose", "hanging-process"],
+      : [process.env.TEST_COMPACT_OUTPUT === "true" ? "basic" : "verbose", "hanging-process"],
```

Comment:
we need a more consistent approach on how we check env variables, something we check strictly for `"true"` and other times we just check if the env variable is set

Reply:
I have no hard opinion about it but what ever we do should do consistently. And if that rule can be enforced by linter  then let it be individual choice.